From fbf33babe3fb52323f098aa300b51bf8fc5ee363 Mon Sep 17 00:00:00 2001
From: Alexey Tikhonov <atikhono@redhat.com>
Date: Wed, 19 May 2021 17:20:52 +0200
Subject: [PATCH] TOOLS: removed unneeded debug message
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This message was logged before `sss_tool_init()` that sets debug level,
thus ignoring configured debug level.

Since the same message is printed via `ERROR` on a next line, this log
message doesn't add any information and can be simply removed.

Reviewed-by: Tomáš Halman <thalman@redhat.com>
---
 src/tools/common/sss_tools.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/tools/common/sss_tools.c b/src/tools/common/sss_tools.c
index 637e251f6..806667f46 100644
--- a/src/tools/common/sss_tools.c
+++ b/src/tools/common/sss_tools.c
@@ -512,7 +512,6 @@ int sss_tool_main(int argc, const char **argv,
 
     uid = getuid();
     if (uid != 0) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Running under %d, must be root\n", uid);
         ERROR("%1$s must be run as root\n", argv[0]);
         return EXIT_FAILURE;
     }
-- 
2.26.3

